whois-client/src/main/java/net/ripe/db/whois/api/rest/domain/ErrorMessages.java

```diff
@@ -9,26 +10,32 @@
 
 import java.util.List;
 
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY;
+
 @XmlRootElement(name = "errormessages")
 @XmlAccessorType(XmlAccessType.FIELD)
+@JsonInclude(NON_EMPTY)
 public class ErrorMessages {
 
     @XmlElement(name = "errormessage")
-    private List<ErrorMessage> errorMessages;
+    protected List<ErrorMessage> errorMessages;
 
     public ErrorMessages(final List<ErrorMessage> errorMessages) {
         this.errorMessages = errorMessages;
     }
 
     public ErrorMessages() {
-        this.errorMessages = Lists.newArrayList();
+        errorMessages = Lists.newArrayList();
     }
-
-    public void addErrorMessage(ErrorMessage error) {
+    public void addMessage(final ErrorMessage error) {
```

Comment:
Why rename these methods? It should be ErrorMessages?

Reply:
I will revert this, probable same left over from a previous logic

Reply:
yes but getter and setter name should have same format

Reply:
yes, probably I renamed that property to Message and then created the getter and setter...then turn back the property to the previous name but forget about the getter and setter (I usually create those method using command+n create Getter / Setter)

Reply:
reverted